### PR TITLE
Dev 12.0 afc 1625b

### PR DESCRIPTION
**Pull request by anderswallenquist**
Some more variants of sync functions

### CODE REVIEW

File: outplacement_crmsync/models/res_partner.py [hunk 1]
```diff
@@ -50,17 +69,143 @@ def get_jobseeker_data(self):
             'res.partner', 'search_read',
                 [
                     # ~ [['social_sec_nr', '=', self.social_sec_nr]],
-                    [[]],
+                    [['id','=',26]],
```
Comment (SebastianGieser-Vertel) [inline outplacement_crmsync/models/res_partner.py:72]: should this be hardcoded? What if there is no partner with id = 26?

File: outplacement_crmsync/models/res_partner.py [hunk 2]
```diff
@@ -50,17 +69,143 @@ def get_jobseeker_data(self):
...
+        Test mot demo-data / utan partner_ssn
+        """
+        
+        xmlrpc = crm_serverII(self.env)
+        
+        
+
+        
+        partner = xmlrpc.common.env['res.partner'].browse(
+            xmlrpc.common.env['res.partner'].search([('id','=',26)],limit=1))
```
Comment (SebastianGieser-Vertel) [inline outplacement_crmsync/models/res_partner.py:174]: Does this work even if there is no partner with id  = 26?

File: sale_outplacement/models/sale.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+class Product(models.Model):
+    _inherit = 'product.template'
+    
+    tlr_ref = fields.Char(string='Tlr ID',)
+    
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+    
+    forvantatantal = fields.Integer(string='Förväntat antal')
```
Comment (SebastianGieser-Vertel) [inline sale_outplacement/models/sale.py:157]: Rename to expected_amount, preferably also change the string accordingly.

File: sale_outplacement/models/sale.py [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+      
+                    })
+                self.env['sale.order.line'].create({
+                    'order_id': self.id,
+                    'product_id': product.id,
+                    'qty': int(artikel['nuvarandeAntal']),
+                    'forvantatantal': int(artikel['forvantatAntal']),
+                    })
+    @api.model
+    def get_order_from_ordertjansten(self):
```
Comment (SebastianGieser-Vertel) [inline sale_outplacement/models/sale.py:58]: this function returns hardcoded test data, maybe rename it to get_order_from_ordertjansten_test

File: sale_outplacement/models/sale.py [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+_logger = logging.getLogger(__name__)
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.one
+    def update_order_from_ordertjansten(self):
+        res = self.env['ipf.showorder.client.config'].get_order_id(self.name)
+        # ~ res = self.env['sale.order'].get_order_from_ordertjansten()
+        for artikel in res.get('artikelList',[]):
```
Comment (SebastianGieser-Vertel) [inline sale_outplacement/models/sale.py:41]: This variable should be in english

File: sale_outplacement/data/product.xml [hunk 6]
```diff
@@ -33,5 +33,16 @@
       <field name="activity_type_id" ref="mail.mail_activity_data_todo" />
       <field name="product_id" ref="product_suborder" />
     </record>
+    
+    
+    <record id="ir_actions_update_order_from_ordertjansten" model="ir.actions.server">
+      <field name="name">Uppdatera_order_från_Ordertjänsten</field>
```
Comment (SebastianGieser-Vertel) [inline sale_outplacement/data/product.xml:39]: This should be in english.

**Review by SebastianGieser-Vertel — CHANGES_REQUESTED** [review]

File: outplacement_crmsync/models/res_partner.py (hunk 1 above)
Comment (anderswallenquist) [reply outplacement_crmsync/models/res_partner.py:72]: yes, i like hard coded test data ;-)

File: outplacement_crmsync/models/res_partner.py (hunk 2 above)
Comment (anderswallenquist) [reply outplacement_crmsync/models/res_partner.py:174]: With demo-data there Brandon Lee is 26

File: sale_outplacement/models/sale.py (hunk 4 above)
Comment (anderswallenquist) [reply sale_outplacement/models/sale.py:58]: Good idea, that means that we ahve to reame all references (i think one server action) too and test it again.

File: sale_outplacement/models/sale.py (hunk 5 above)
Comment (anderswallenquist) [reply sale_outplacement/models/sale.py:41]: True

File: sale_outplacement/data/product.xml (hunk 6 above)
Comment (anderswallenquist) [reply sale_outplacement/data/product.xml:39]: True

**Comment by anderswallenquist** [comment]
Please write a Jira with your comments

**Comment by arturchilangwa-vertel** [comment]
There is no buggs the code, so I am merging this.

**Pull request merged**